.buildkite/pipeline.yml

```
@@ -102,6 +95,7 @@ steps:
         plugins: [$CI_TOOLKIT]
         command: |
           install_gems
+          install_swiftpm_dependencies --project "$DEMO_PROJECT"
```

As a result of all that, one could also ponder if it's that worth it to declare `DEMO_PROJECT` as an env var given that the only place where you end up referencing that env var is in a single step of the `pipeline.yml` file and nowhere else…

So maybe it'd be simpler and enough to replace this line with this
```suggestion
          install_swiftpm_dependencies --project "Demo/Gravatar-Demo.xcodeproj"
```
And then delete it from the `shared-pipeline-vars` as well as from line 11 that added it to the `env:` attribute?

---

Yeah, that's fair.  I was trying to keep our hardcoded values in one place.  But if we change this hardcoded value in the future, we'll have to update this path in several places anyway.  I'll simplify.

On the plus side, it will read beautifully this way.